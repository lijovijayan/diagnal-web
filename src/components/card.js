import { LazyImage } from './lazy.image'
export function Card(props) {
    const { imageURL, name } = props;
    return (
        <div className="movie-card flex flex-col xl:mb-6 lg:mb-6 md:mb-5 mb-4  xl:p-3 lg:p-3 md:p-2 p-2">
            {/* mb-6 */}
            <LazyImage src={imageURL} alt={name} className={'mb-1'} />
            <div>{name}</div>
            <div className="play-button">
                <div className="play"></div>
            </div>
        </div>
    )
}
