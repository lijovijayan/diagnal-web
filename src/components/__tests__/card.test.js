import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import { Card } from '../card'

describe('testing card', () => {
    it('rendered properly', () => {
        const component = shallow(<Card imageURL={''} name={''} />)
        const tree = toJson(component)
        expect(tree).toMatchSnapshot()
    })
})
