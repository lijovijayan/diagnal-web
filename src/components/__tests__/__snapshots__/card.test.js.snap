// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing card rendered properly 1`] = `
<div
  className="movie-card flex flex-col xl:mb-6 lg:mb-6 md:mb-5 mb-4  xl:p-3 lg:p-3 md:p-2 p-2"
>
  <LazyImage
    alt=""
    className="mb-1"
    src=""
  />
  <div />
  <div
    className="play-button"
  >
    <div
      className="play"
    />
  </div>
</div>
`;
